src/components/charts/ChartHeaderFilters.tsx

```diff
@@ -65,44 +65,63 @@ function ChartHeaderFilter({ selectedChartRepo, handleCloseFilter, includeDeprec
             history.push(`${url}?${qs}`);
         }
     }
-   
+
+    const setStore = (event) => {
```

Comment:
try to add the return type of the function as well. const setStore = (event): void => {

Reply:
Done